common-arcana.lic

```diff
@@ -166,11 +166,6 @@ module DRCA
   end
 
   def release_cyclics
-    get_data('spells')
-      .spell_data
-      .select { |_name, properties| properties['cyclic'] }
-      .select { |name, _properties| DRSpells.active_spells.keys.include?(name) }
-      .map { |_name, properties| properties['abbrev'] }
-      .each { |abbrev| fput("release #{abbrev}") }
+    bput('release cyclic')
```

Comment:
I guess I've never noticed fput lose to roundtime, just retry until successful.  

Every spell has a different release message.  I wonder if bput('release cyclic',/.*/) would work.  Will give it a test